src/Domain/Structure/ElementId.cpp

```diff
@@ -130,6 +130,21 @@ std::array<SegmentId, VolumeDim> ElementId<VolumeDim>::segment_ids()
   }
 }
 
+SegmentId segment_id(const size_t dim) const noexcept {
```

Comment:
accessor should be in header

Reply:
same question: does it inline automatically?